wpeverest/everest-forms
# Feature - Email field mailcheck feature to offer additional controls

### All Submissions:






### Changes proposed in this Pull Request:



This PR aims to reduce misspelled email addresses in Everest Forms using the [Mailcheck](https://github.com/mailcheck/mailcheck
) JS library. Thus, the email bounce rate will be lower. Additionally `everest_forms_mailcheck_domains` and `everest_forms_mailcheck_toplevel_domains` filter hooks are available to modify for users.

**To disable this feature:**
If the user wants to disable it then they can use filter hook to completely off this feature which will not load any external JS library.
```
add_filter( 'everest_forms_mailcheck_enabled', '__return_false' );
```

### How to test the changes in this Pull Request:

1. Add the email field and preview the form.
2. Use `example@gmail.con` and click outside the email field.
3. Confirm if you see the suggestion message. If seen, you are good to go :)

### Types of changes:




### Other information:




### Changelog entry

> Feature - Email field Mailcheck feature to offer additional controls.
